src/policy/fees.cpp

```diff
@@ -579,11 +574,26 @@ CBlockPolicyEstimator::CBlockPolicyEstimator(const fs::path& estimation_filepath
 
 CBlockPolicyEstimator::~CBlockPolicyEstimator() = default;
 
-void CBlockPolicyEstimator::processTransaction(const CTxMemPoolEntry& entry, bool validFeeEstimate)
+void CBlockPolicyEstimator::TransactionAddedToMempool(const NewMempoolTransactionInfo& tx, uint64_t mempool_sequence)
```

Comment:
Nit: Remove the unused argument names in the interface functions?

Reply:
Yes, I was only suggesting to remove the names, e.g. change `uint64_t mempool_sequence` to `uint64_t /*unused*/`.

Reply:
thanks I understand,  updated